src/seq/bioseq.jl

```diff
@@ -63,20 +63,48 @@ typealias CharSequence      BioSequence{CharAlphabet}
 
 remove_newlines(s) = replace(s, r"\r|\n", "")
 
-macro dna_str(seq, flags...)
-    return DNASequence(remove_newlines(seq))
+macro dna_str(seq, flag)
```

Comment:
Sounds good, 😄  I'll add those in an hour or so - moving to a new office and desk :+1: